core/test-runtime/src/lib.rs

```diff
@@ -459,9 +467,14 @@ cfg_if! {
 				}
 			}
 
-			impl consensus_aura::AuraApi<Block, AuraId> for Runtime {
+			impl consensus_aura::AuraApi<Block, AuraId, sr25519::Signature> for Runtime {
 				fn slot_duration() -> u64 { 1 }
 				fn authorities() -> Vec<AuraId> { system::authorities() }
+				fn construct_equivocation_report_call(
+					_proof: AuraEquivocationProof<<Block as BlockT>::Header,sr25519::Signature>
+				) -> Vec<u8> {
+					vec![]
```

Comment:
I was wondering why this was empty, then realized that this is the _test_ runtime.